shell.nix

```diff
@@ -1,40 +1,26 @@
-{ nixpkgs ?  import (builtins.fetchTarball "https://github.com/NixOS/nixpkgs/archive/13e0d337037b3f59eccbbdf3bc1fe7b1e55c93fd.tar.gz") { } }:
+{ pkgs ? import
+    (
+      builtins.fetchTarball "https://github.com/NixOS/nixpkgs/archive/13e0d337037b3f59eccbbdf3bc1fe7b1e55c93fd.tar.gz"
+    )
+    { }
+}:
 
 let
-  js2nix = nixpkgs.callPackage ./default.nix { };
-  tree = js2nix.load ./yarn.lock {
-    overlays = [
-      (self: super: {
-        "babel-jest@27.0.2" = super."babel-jest@27.0.2".override
-          # Fix peer dependencies
-          (x: { modules = x.modules ++ [ (self."@babel/core@7.14.3") ]; });
-      })
-    ];
+  js2nix = pkgs.callPackage ./default.nix { };
+  env = js2nix {
+    package-json = ./package.json;
+    yarn-lock = ./yarn.lock;
   };
-
-  devNodeModules = js2nix.makeNodeModules ./package.json {
```

Comment:
So we dont need to differentiate between `dev` and `prod` anymore? It's just the full set available?

Reply:
We don't have to differentiate but just pick up the closure we need, like `dev` or `prod`, but:
```nix
let
  env = js2nix {
    # ...
  };
in env.nodeModules # dev, by default. Or `in env.nodeModules.prod`
```